src/main/kotlin/three/consulting/epoc/dto/FirebaseUserDTO.kt

```diff
@@ -0,0 +1,8 @@
+package three.consulting.epoc.dto
+
+import javax.validation.constraints.NotBlank
+
+data class FirebaseUserDTO(
+    @field:NotBlank val role: String,
+    @field:NotBlank val firebaseUid: String
+)
```

Comment:
This feels like a redundant class, since the fields already exist in `EmployeeDTO` can't you just use that? This is already used in `EmployeeController` so feels natural it would just be the employee given as the request

Reply:
I would like to require _role_ and _firebaseUid_ values with the endpoint - these are optional in _EmployeeDTO_ class. Also there are some field that are unnecesary in this particular operation. However, we could create a new FirebaseController. On the other hand, if it has only one endpoint it might be unnecessary.

Reply:
They can stay as optional for now, but they should be mandatory for employee as well. The problem is we need to run the current migrations to production first. We can make new migrations making the uid and role mandatory in a future pr